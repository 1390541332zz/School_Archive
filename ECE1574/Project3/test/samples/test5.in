File: test/maps/catabaw.asc
Greatest Inclination 6