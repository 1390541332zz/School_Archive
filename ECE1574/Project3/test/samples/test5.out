Loading: test/maps/catabaw.asc
Number of rows: 15
Number of columns: 20
	         0    1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16   17   18   19
	    0  682  688  731  771  799  756  770  786  771  791  937 1034  910  714  616  641  791  789  973 1004
	    1  676  744  786  837  800  753  740  779  909  966  846  692  628  688  772  734  750  900  948  719
	    2  726  786  771  784  782  802  860  950  863  708  655  701  783  820  947  962  920  804  631  551
	    3  760  740  777  842  913  920  816  701  680  707  758  868  893  996  922  760  638  569  530  512
	    4  814  897 1005  958  821  742  687  743  818  931  922  948  866  741  662  557  519  508  525  523
	    5 1098 1058  910  787  709  681  736  908 1020  964  826  710  645  590  532  531  586  639  672  726
	    6 1081 1053  901  750  727  740  836  865  774  714  736  634  556  559  637  724  781  790  779  764
	    7  944  929  911  912  875  804  701  642  646  696  572  582  684  777  794  783  799  818  827  860
	    8  950 1002 1070  960  767  635  608  588  614  585  672  782  801  772  762  792  896  949  939  846
	    9 1164 1112  922  735  658  645  609  574  620  731  812  751  739  748  793  885  783  664  584  521
	   10 1144  936  700  640  623  630  665  692  789  784  739  733  765  847  860  682  535  483  478  531
	   11 1071  784  705  782  865  903  840  814  740  718  725  821  901  779  604  511  498  508  602  609
	   12  897  822  888  978 1029  831  710  705  707  736  875  824  641  542  511  577  721  745  654  678
	   13  837  744  701  842  929  720  674  702  793  865  745  582  532  547  646  780  729  666  690  675
	   14  668  662  768  770  672  706  797  827  721  606  567  584  666  759  768  675  647  647  654  639
	
Command: Greatest Inclination
	Height change:  -77
	Steps:          24
	Average change: -3.20833
	Starting row:   6
Number of rows: 15
Number of columns: 20
	     0  1  2  3  4  5  6  7  8  9 10 11 12 13 14 15 16 17 18 19
	 0   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  >  X
	 1   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  /  -  -
	 2   -  -  -  -  -  -  -  -  -  -  -  -  -  -  >  >  /  -  -  -
	 3   -  -  -  -  -  -  -  -  -  -  -  -  >  /  -  -  -  -  -  -
	 4   -  -  -  -  -  -  -  -  -  >  >  /  -  -  -  -  -  -  -  -
	 5   >  v  -  -  -  -  -  >  >  ^  -  -  -  -  -  -  -  -  -  -
	 6   ^  v  -  -  -  -  /  -  -  -  -  -  -  -  -  -  -  -  -  -
	 7   -  \  -  >  >  /  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 8   -  -  >  ^  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 9   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	10   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	11   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	12   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	13   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	14   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	
